.github/workflows/CD.yml

```diff
@@ -0,0 +1,91 @@
+name: CD
+
+on:
+  pull_request:
+    types: [closed]
```

Comment:
I am not sure closed is appropriate as it may not have been merged, I have my personal deployments set on push, master, either with an if inside a job or on for the entire file:
`
on:
  push:
    branches:
      - master

if: github.event_name == 'push'
needs: push
`

Reply:
Sorry I made this confusing, I think on needs the push with branch master. At the moment it will only run on a pull_request event type and then we check if github.event_name (which can only ever be pull request) is equal to push.

Reply:
No, it's my fault. I wanted to avoid the CD workflow running twice when a PR is merged into master (once for the PR event, and another for the push event), but I had a brainfart and messed up the job conditional. I think this should do it.

It won't run for direct pushes to master, but I don't think we should be doing that anyway. All changes should come in the form of merged PRs. If Zak needs to urgently push a change to master, he can deploy the frontend manually anyway.